artview/components/level.py

```diff
@@ -0,0 +1,218 @@
+"""
+level.py
+
+Class instance used for modifying level via Display window.
+"""
```

Comment:
it is not much, but we have this [wording](https://github.com/gamaanderson/artview/blob/grid/artview/core/core.py#L56-L58)